lib/viberroo/bot.rb

```diff
@@ -140,8 +140,11 @@ def remove_webhook
     # @see https://viber.github.io/docs/tools/keyboards/#buttons-parameters
     # @see https://developers.viber.com/docs/api/rest-bot-api/#keyboards
     #
-    def send_message(message, keyboard: {})
-      request(URL::MESSAGE, { receiver: @response.user_id }.merge(message, keyboard))
+    def send_message(message:, keyboard: {})
+      request(
+        URL::MESSAGE,
+        { receiver: @response.user_id }.merge(message, keyboard, min_api_version_hash(keyboard))
```

Comment:
Someone might already be using `min_api_version` key as part of the `message` on `v0.3.5`. This will sneakily override the `min_api_version` which will introduce a subtle bug. Also, this will take away the control of using `min_api_version` from the client application.
 
 Let's have something that allows the client application to override the `min_api_version` e.g.:
 ```ruby
      request(
        URL::MESSAGE,
        { receiver: @response.user_id }.tap do |hash|
          hash.merge!(message, keyboard)
          if hash[:min_api_version].nil?
            hash[:min_api_version] = min_api_version(keyboard)
          end
        end
      )
 ```